go.mod

```diff
@@ -4,11 +4,16 @@ go 1.12
 
 require (
 	github.com/Azure/go-autorest v11.1.2+incompatible // indirect
+	github.com/Masterminds/glide v0.13.2 // indirect
```

Comment:
Why do we have a dependency on this? :open_mouth:

Reply:
go removed after running `go mod tidy`